### DISCUSSION TITLE
Мои наблюдения по опыту использования Обсидиана | Paperless Forest

### DISCUSSION THREAD

**Discussion by giscus** [anareaty/paperless-forest-old#265] [General]
# Мои наблюдения по опыту использования Обсидиана | Paperless Forest

Я использую Обсидиан уже два года, и постоянно пробую в нём что-то новое. В этом посте буду собирать список рандомных заметок о том, что для меня работает, а что нет, и других кратких наблюдений по опыту использования.
 Дашборды красивые, я сделала несколько очень крутых дашбордов, и с датавью, и попроще, но в итоге я их практически никогда не использую. Для навигации чаще всего использую поиск по названию и алиасам. Можно делать это с помощью Quick Switcher-а но я чаще использую поисковую строку в плагине Home tab.

https://paperless-forest.ru/obsidian-random-notes/

**Comment by TheDiVaZo** [comment]
Интересные наблюдения, спасибо

**Reply by anareaty** [reply]
Спасибо) Думаю, этот список будет дополняться со временем.

**Comment by pav18** [comment]
Спасибо за ваши знания и советы. Зашел в статью про кастомные чекбоксы, но пришлось задержаться ;)
Два года не использовал теги - не встраивались в систему. А тут начал. Отмечаю книги в заметках, Софт и еще много чего. При этом Маркеры могут быть (в зависимости от необходимости) разные. Например Книги - #book   [[Books]]  Books::   - [r] Книга      Есть dataviewjs  который все это собирает при желании в одну таблицу.      Например по тексту удобнее книгу записать на отдельную строчку и отмаркировать тегом или таском.
